packages/react-core/src/components/DataList/DataListToggle.tsx

```diff
@@ -21,29 +21,34 @@ export interface DataListToggleProps extends React.HTMLProps<HTMLDivElement> {
   'aria-controls'?: string;
 }
 
-export const DataListToggle: React.FunctionComponent<DataListToggleProps> = ({
+/**
```

Comment:
What's the value in adding so many instances of this empty comment block? If it's only to satisfy eslint, we should handle it another way. We could either actually add text to these comments so they serve a purpose, or we could exclude this from being something that eslint requires. I think either way would be better than bloating the codebase with the empty comments.